NEWS.md

```diff
@@ -242,6 +242,10 @@ to specify the unit of the input age (#569)
 
 - The `start_date` and `end_date` parameters of `derive_var_ady()`, `derive_var_aendy()` and `derive_var_astdy()` have been renamed to `reference_date` and `date`, respectively (#121)
 
+- The `derive_last_dose()` function has been split into a general function `derive_vars_last_dose()`,
```

Comment:
See above

Reply:
Sorry about this, I moved the updates into the "admiral development version".